core/tests/webdriverio_desktop/contributorAdminDashboard.js

```diff
@@ -350,4 +350,90 @@ describe('Contributor Admin Dashboard', function () {
     await contributorDashboardAdminPage.expectNoStatsElement();
     await users.logout();
   });
+
+  it(
+    'should be able to filter translation submissions using time range' +
+      ' filter',
+    async function () {
+      await users.login(TRANSLATION_COORDINATOR_EMAIL);
+      await contributorDashboardAdminPage.get();
+
+      await contributorDashboardAdminPage.navigateToTranslationSubmitterTab();
+      await contributorDashboardAdminPage.waitForLoadingMessageToDisappear();
+
+      await contributorDashboardAdminPage.switchLanguage('Albanian (shqip)');
+      await contributorDashboardAdminPage.waitForLoadingMessageToDisappear();
+      await contributorDashboardAdminPage.expectStatsElementCountToBe(1);
+
+      await contributorDashboardAdminPage.setFirstDatePickerValue(
+        new Date(new Date().getTime() - 24 * 60 * 60 * 1000)
+      );
+      await contributorDashboardAdminPage.waitForLoadingMessageToDisappear();
+      await contributorDashboardAdminPage.expectNoStatsElement();
+
+      await users.logout();
+    }
+  );
+
+  it(
+    'should be able to filter translation reviewers using time range' +
+      ' filter',
+    async function () {
+      await users.login(TRANSLATION_COORDINATOR_EMAIL);
+      await contributorDashboardAdminPage.get();
+
+      await contributorDashboardAdminPage.navigateToTranslationReviewerTab();
+      await contributorDashboardAdminPage.waitForLoadingMessageToDisappear();
+
+      await contributorDashboardAdminPage.switchLanguage('Albanian (shqip)');
+      await contributorDashboardAdminPage.waitForLoadingMessageToDisappear();
+      await contributorDashboardAdminPage.expectStatsElementCountToBe(1);
+
+      await contributorDashboardAdminPage.setFirstDatePickerValue(
+        new Date(new Date().getTime() - 24 * 60 * 60 * 1000)
+      );
```

Comment:
I think there are some other situations that need to be tested. For example:

* Using the first date filter
  * An element matches the date filter
* Using the last date filter
  * An element matches the date filter
  * No elements match the date filter
* Using both filters
  * An element matches the filters
  * No elements match the filter because elements are earlier than the filtered range
  * No elements match the filter because elements are later than the filtered range
  * No elements match the filter because elements are on both sides of the filtered range (might be able to skip this one)

Reply:
Hello @U8NWXD,

1.  Using the first date filter
- By default, first date filter is set to `today's date`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches first date filter.

2. Using the last date filter
- By default, last date filter is set to `today's date - 90 days`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches last date filter.
- In this feature users are not allowed to choose any future dates and when we create any submissions or reviews, its created in last 24 hours.

3. Using both filters
- By default, first date filter is set to `today's date` and last date filter is set to `today's date - 90 days`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches first as well as last date filter.
- When set first date filter to yesterday's date that is the case when no elements match the filter because elements are earlier than the filtered range.
- In this feature users are not allowed to choose any future dates and when we create any submissions or reviews, its created in last 24 hours.
- In this feature users are not allowed to set first date before last date. So, this case won't occur.

All the fore mentioned cases have been tested in unit tests, respectively. Since, we are moving towards acceptance test, this test tests the usual userflow.

Reply:
Hello @U8NWXD,

1.  Using the first date filter
- By default, first date filter is set to `today's date`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches first date filter.

2. Using the last date filter
- By default, last date filter is set to `today's date - 90 days`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches last date filter.
- In this feature users are not allowed to choose any future dates and when we create any submissions or reviews, its created in last 24 hours.

3. Using both filters
- By default, first date filter is set to `today's date` and last date filter is set to `today's date - 90 days`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches first as well as last date filter.
- When set first date filter to yesterday's date that is the case when no elements match the filter because elements are earlier than the filtered range.
- In this feature users are not allowed to choose any future dates and when we create any submissions or reviews, its created in last 24 hours.
- In this feature users are not allowed to set first date before last date. So, this case won't occur.

All the fore mentioned cases have been tested in unit tests, respectively. Since, we are moving towards acceptance test, this test tests the usual userflow.

Reply:
Hello @U8NWXD,

1.  Using the first date filter
- By default, first date filter is set to `today's date`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches first date filter.

2. Using the last date filter
- By default, last date filter is set to `today's date - 90 days`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches last date filter.
- In this feature users are not allowed to choose any future dates and when we create any submissions or reviews, its created in last 24 hours.

3. Using both filters
- By default, first date filter is set to `today's date` and last date filter is set to `today's date - 90 days`. So, the initially when we are checking whether the stats element appear is actually the case when an element matches first as well as last date filter.
- When set first date filter to yesterday's date that is the case when no elements match the filter because elements are earlier than the filtered range.
- In this feature users are not allowed to choose any future dates and when we create any submissions or reviews, its created in last 24 hours.
- In this feature users are not allowed to set first date before last date. So, this case won't occur.

All the fore mentioned cases have been tested in unit tests, respectively. Since, we are moving towards acceptance test, this test tests the usual user flow.

Reply:
Ahh, I didn't appreciate that we're limited to testing with events within the last 24 hours because of how the tests work. In that case, I think the tests you've written are the best we can do